zarnitsa/series/DataAugmenterTimeSeries.py

```diff
@@ -60,20 +60,18 @@ def data_interpolation(
         Interpolate pandas column
         filling the gaps
         """
-        if len(col) == 0:
+        if not col.shape[0]:
             raise ValueError(f"Iterable object <{type(col)}> is empty! Check input!")
-
+        if not col[col.isna()].shape[0]:
+            return col
         if limit is None:
             limit = int(col[col.isna()].shape[0] * freq)
 
-        if method in ["fillna"]:
+        if method in ("fillna"):
```

Comment:
if you want to use it as a tuple, you should add comma:
`("fillna",)`  otherwise it is a string with redundant brackets, but it is not important

Reply:
oh, yeah
you right, my bad
I will fix it